newrelic/resource_newrelic_alert_plugins_condition_test.go

```diff
@@ -0,0 +1,513 @@
+package newrelic
+
+import (
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccNewRelicAlertPluginsCondition_Basic(t *testing.T) {
+	rName := acctest.RandString(5)
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckNewRelicAlertPluginsConditionDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccCheckNewRelicAlertPluginsConditionConfig(rName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckNewRelicAlertPluginsConditionExists("newrelic_alert_plugins_condition.foo"),
+					resource.TestCheckResourceAttr(
+						"newrelic_alert_plugins_condition.foo", "name", fmt.Sprintf("tf-test-%s", rName)),
+					resource.TestCheckResourceAttr(
```

Comment:
This check for `type` should be deleted, since it is not a valid attribute for this resource.